css/style.css

```diff
@@ -1 +1,1091 @@
-body {}
\ No newline at end of file
+:root {
+  --btn-primary-color: rgba(33, 231, 134, 1);
+  --secondary-color: rgba(20, 27, 34, 1);
+  --article-bg-color: rgba(255, 255, 255, 0.05);
+  --main-background-color: rgb(13, 13, 17);
+  --text-color: rgba(194, 195, 197, 1);
+  --inactive-color: rgba(136, 139, 142, 1);
+  --text-font: "Poppins", sans-serif;
+}
+
+/* Uneversal css */
+
+* {
+  padding: 0;
+  margin: 0;
+  background-color: rgba(4, 11, 17, 1);
```

Comment:
Translate global text and background styles from the universal selector * to the body selector:

Why: Using the universal selector * can negatively impact performance and can inadvertently apply styles to all elements, including those you may not intend to style. Applying general styles directly to the body is more efficient and ensures that only the base of your document is affected.

[Read more about CSS performance and the universal selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Universal_selectors).

Reply:
Thank you for this comment. Now I see how * can impact whole code. 

#fixed